pheweb/load/phenolist_gbma.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env/python3
+
+import sys
+import json
+import gzip
+import glob
+import os
+import json
+import functools
+
+def run():
+
+    with open(sys.argv[1]) as f:
+        phenolist = json.load(f)
+
+    pheno2conf = {}
+    for file in glob.glob('/mnt/nfs/pheweb/gbma/conf/*.json'):
+        with open(file, 'rt') as f:
+            conf = json.load(f)
+        pheno = os.path.basename(file).replace('_conf.json', '_meta_pheweb')
+        pheno2conf[pheno] = conf['meta']
+
+    for pheno in phenolist:
+        if pheno['phenocode'] == 'AAA_meta_pheweb':
+            pheno['phenostring'] = 'Abdominal aortic aneurysm'
+        if pheno['phenocode'] == 'Asthma_meta_pheweb':
+            pheno['phenostring'] = 'Asthma'
+        if pheno['phenocode'] == 'IPF_meta_pheweb':
+            pheno['phenostring'] = 'Idiopathic pulmonary fibrosis'
+        if pheno['phenocode'] == 'POAG_meta_pheweb':
+            pheno['phenostring'] = 'Primary open-angle glaucoma'
+        if pheno['phenocode'] == 'ThC_meta_pheweb':
+            pheno['phenostring'] = 'Thyroid cancer'
+        if pheno['phenocode'] == 'UtC_meta_pheweb':
+            pheno['phenostring'] = 'Uterine cancer'
```

Comment:
Instead of using a large if-structure, one option would be to use a dictionary to store all of the different values.
Like so:
```python
phenostring_values={'AAA_meta_pheweb':'Abdominal aortic aneurysm',
'Asthma_meta_pheweb':'Asthma'
#add values as necessary
}
try:
    pheno['phenostring'] = phenostring_values[pheno['phenocode']]
except:
    pheno['phenostring'] = "NA" #default value for not implemented phenocodes
```

IMO it makes it simpler to add different values.

Reply:
Totally agree, this was a quick one-off ad-hoc for getting the pheno names for the pilot browser - the mappings could be in a text file too. Like Mitja pointed out, questionable if these should be in the repo in this way but at least there's a track